src/views/Viewer.vue

```diff
@@ -26,7 +26,7 @@
 		v-if="initiated || currentFile.modal"
 		id="viewer"
 		:class="{'icon-loading': !currentFile.loaded && !currentFile.failed}"
-		:clear-view-delay="isTesting ? -1 : 5000 /* prevent cypress timeouts */"
+		:clear-view-delay="isTesting ? -1 : -1"
```

Comment:
```suggestion
		:clear-view-delay="-1"
```

But it would be nice to keep it that way if this line will be changed in the near future. In that case, let's keep the comment too.

Reply:
So would this your recommendation basically?
```suggestion
		:clear-view-delay="isTesting ? -1 : -1 /* prevent cypress timeouts */""
```

Reply:
Haha comment looks indeed inadequate.

```suggestion
		:clear-view-delay="-1" /* In case of modification, check if testing to prevent cypress timeouts: "isTesting ? -1 : ..." */
```

Again, this only makes sense if the line has a chance of getting changed in the future to not forget that hack.

Reply:
Lets wait with that until we get some more feedback on this issue :)

Reply:
will do it differently